### PR TITLE
Massive overhaul to main faction loadouts, new weapons, new sprites, timelocks and job flavors

### PR DESCRIPTION
**Pull request by Legate-Maxson**
These need to be tested on the server.

Changes the slots as requested by Coders and staff.

Rebalances some equipment. 

Adds new equipment.

Changes loadouts for many jobs. 

___________________________________

Currently missing or needed to be reworked at a later date.

- Town: New vision for town changes on this PR were removed and saved to be worked on later. 
Status: low priority

-Enclave: Weapons and what they want to be need to be discussed with lore heads and coders. Weapons need massive updates. Status: medium

Several weapon sprites need to be updated, and several weapons may need further balance inspection.
Status: It is what it is.

### CODE REVIEW

**Comment by BadAtThisGame302** [comment]
Who let Raptor cook!!! He is BoS biased bruh shut this shit down!!

**Comment by Stutternov** [comment]
> Who let Raptor cook!!! He is BoS biased bruh shut this shit down!!

He won't give scribe the instagib rifle..... it's so joever......

File: code/modules/projectiles/guns/ballistic/enc_minigun.dm [hunk 1]
```diff
@@ -115,7 +115,7 @@
 	automatic = 1
 	autofire_shot_delay = 1
 	burst_shot_delay = 1
-	fire_delay = 1
+	fire_delay = 2
```
Comment (mazzinsanity) [inline code/modules/projectiles/guns/ballistic/enc_minigun.dm:118]: This won't work. If you want to make it slower, you need to alter the autofire_shot_delay field

**Comment by ApplePlastic** [comment]
Khanjak here. A little Mongolpilled. 

I think Loadout 3 for the senior enforcer (the one with the sniper rifle and one magazine? lol) is incredibly niche compared to the other two, which are both weirdly barren in terms of equipment, but they're both fairly good weapons in terms of practicality. I suggest replacing the 10mm SMG or the sniper rifle with the riot shotgun, excluding the three boxes of train-shot since that ammo-type's pretty bad when used in the riot shotgun anyway.

**Comment by Laus4Deus** [comment]
Could this PR be broken down into smaller ones?

**Review by maaacha — APPROVED** [review]
FINALLY,,,, FINISHED,,,,,

**Pull request merged**